Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -108,16 +109,30 @@ def get_incident_labels_map(labels):
     return labels_map
 
 
+def build_similar_keys_list(similar_incident_keys):
+    similar_keys_list = []
+    for key, value in similar_incident_keys.items():
+        if isinstance(value, int):
+            value = str(value).replace('"', r'\"').replace("\n", "\\n").replace("\r", "\\r")
+            compare_str = '{}:="{}"'
+            similar_key = compare_str.format(key, value)
+            similar_keys_list.append(similar_key)
+
+        else:
+            value = value.replace('"', r'\"').replace("\n", "\\n").replace("\r", "\\r")
+            value = value.encode('utf-8')
```

Comment:
1. Can you do this line before the previous one?
If so, we have 4 lines here that are exactly the same, and can be done once.

2. Why don't we solely change the comparison string based on the type?

Reply:
1. no, Right now all the lines are different from each other.
2. done